### ISSUE TITLE
Feature Request - partial entity Update

### ISSUE THREAD

**Issue by fenixil** [tiagoln/BulkExtensions#43]
In some cases we don't need to update all fields of the entity and only few of them instead.

In [SqlBulkTools](https://github.com/olegil/SqlBulkTools#bulkupdate), we could explicitly configure which columns we are going to update.

The idea of current feature request is to support something similar - bulk update only specific fields of the entity.

**Comment by tiagoln** [comment]
Yes, I planed to support this feature in a similar way in the future.
Before I went absent this past couple of months I was working on another feature that I thought would be nice, so I think I'll wrap up that feature and release a beta version to see if it will work as intended, and then I'll begin work on this feature.
